src/azul/indexer/document.py

```diff
@@ -558,7 +592,10 @@ def field_types(cls, field_types: FieldTypes) -> FieldTypes:
         return {
             **super().field_types(field_types),
             'num_contributions': pass_thru_int,
-            'bundles': pass_thru_jsons
```

Comment:
Good thought, although the plurality is actually something not needed in this case. The primary purpose of the `FieldTypes` structure is so that `Document.translate_fields` can replace `None` values within a document so that ElasticSearch will include the field in the index. During this translation process `Document.translate_fields` traverses the document and the `FieldTypes` structure in parallel. When a list value in the document is encountered, `translate_fields` is recursively called within a list comprehension and the current `FieldTypes` structure is simply passed as-is to each call.

This lack of a list within the `FieldTypes` structure is also present in `BaseTransformer._project_types` which has a property `contributors` with value `cls._contact_types()` that gives back a dictionary even though the field `contents.projects.contributors` is a list of dictionaries.